core/storage/email/gae_models.py

```diff
@@ -384,27 +386,28 @@ def get(cls, user_id, exploration_id, thread_id, strict=True):
                 instance id exists in the datastore. Default is True.
 
         Returns:
-            An entry instance that corresponds to the given instance id if it is
-                present in the datastore. Otherwise, None.
+            A FeedbackEmailReplyToIdModel instance that corresponds to the given
+                instance id if it is present in the datastore. Otherwise, None.
         """
         instance_id = cls._generate_id(user_id, exploration_id, thread_id)
         return super(
             FeedbackEmailReplyToIdModel, cls).get(instance_id, strict=strict)
 
     @classmethod
     def get_multi_by_user_ids(cls, user_ids, exploration_id, thread_id):
-        """Returns the entries corresponding to the given user ids in the dict
-        format.
+        """Returns the FeedbackEmailReplyToIdModel instance corresponding to the
+        given user ids in dict format.
 
         Args:
             user_ids: list(str). A list of user ids.
             exploration_id: str. The exploration id.
             thread_id: str. The thread id.
 
         Returns:
-            dict. The enteries correponding to the given list of user ids in
-                dict format having key as the unique user id and value as the
-                entery corresponding to the user id in the key.
+            dict. The FeedbackEmailReplyToIdModel instances corresponding to the
```

Comment:
> Done. I've rechecked from 'get_multi' method of BaseModel class in gae_models.py. One instance corresponds to one unique user ID. Thanks!

I'm not sure if the logic of this method is being misunderstood, but I think that your docstring is still unclear as written. In general, there can be multiple FeedbackEmailReplyToIdModel instances corresponding to the same user ID, since the exp ID and thread ID also need to be taken into account.

When you say "the FeedbackEmailReplyToIdModel instance corresponding to the given user ids" here and above, it suggests that there's just one such model for a bunch of user IDs. Even if you meant "instances" then that still suggests that there's a one-to-one mapping of user_id to FeedbackEmailReplyToIdModel. This leaves unanswered the question of what role the exploration_id and thread_id arguments play, and how they affect the return value.

/cc @prasanna08